tests/kafkatest/tests/copycat_distributed_test.py

```diff
@@ -68,7 +71,7 @@ def test_file_source_and_sink(self, converter="org.apache.kafka.copycat.json.Jso
         # do rebalancing of the group, etc, and b) without explicit leave group support, rebalancing takes awhile
         for node in self.cc.nodes:
             node.account.ssh("echo -e -n " + repr(self.FIRST_INPUTS) + " >> " + self.INPUT_FILE)
-        wait_until(lambda: self.validate_output(self.FIRST_INPUT_LIST), timeout_sec=120, err_msg="Data added to input file was not seen in the output file in a reasonable amount of time.")
+        wait_until(lambda: self._validate_file_output(self.FIRST_INPUT_LIST), timeout_sec=120, err_msg="Data added to input file was not seen in the output file in a reasonable amount of time.")
```

Comment:
It is. This could probably be dropped to be lower now, though I'd rather keep them rather liberal to avoid an spurious errors just due to timing issues.

The problem with Copycat startup in particular is that in distributed mode it may need to join the worker group which has a session timeout of 30s and any sink tasks that start also have a potential 30s timeout. This is also related to https://issues.apache.org/jira/browse/KAFKA-2713. Now that leave group has been implemented, this *should* be ok at much lower value since this class assumes clean bouncing of nodes.

For now, I think I'd like to leave the timeouts relatively high and avoid spurious failures that are just due to being too conservative. I think sometime after the 0.9.0.0 release, we're going to want to do a general cleanup of things like timeouts and get a better handle on system test runtimes.